### DISCUSSION TITLE
how to access GitHub Education, the developer pack

### DISCUSSION THREAD

**Discussion by CoordAcademicaKC** [community/community#110411] [GitHub Education]
### Select Topic Area

Question

### Body

Hello, thank you for your previous response. At our KeepCoding school, we're interested in learning how to access GitHub Education, the developer pack, and whether using this feature would incur any costs. Thank you.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @CoordAcademicaKC , we're happy you're here! You are more likely to get a useful response if you are posting your questions in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by Lanrad23** [comment]
First, check if your school and its students can use GitHub Education. GitHub usually needs students to be at least 13 years old and take classes that lead to a degree or diploma. Teachers and school staff can apply for benefits, too. After you apply, you'll need to wait a bit for GitHub to check if you're really a student or teacher. This wait can be short or long, but GitHub tends to be fast with these checks.

**Comment by coder-ralph** [comment]


**Reply by queenofcorgis** [reply]
@coder-ralph 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by samus-aran** [comment]
Hi @CoordAcademicaKC 

You can ask your institution to join the GitHub Campus Program by [applying here](https://education.github.com/schools). 👈

Hope that helps 👍

**Reply by samus-aran** [reply]
Re-reading the request:

You are asking how to apply as a student or faculty? You may apply with KeepCoding school but you will be asked additional information on that school. This can be done via the application itself.
